_posts/2015-08-19-How-To-Fork-A-Repo-And-Contribute.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout:     post
+title:      "How To Fork A Repo And Contribute"
+subtitle:   "A how to guide for forking a github repo and contribute to an open source project."
+date:       2015-08-19 19:00:00
+author:     "Colin Rubbert"
+header-img: "img/chem.png"
+---
+
+{% include youtubePlayer.html id="G1I3HF4YWEw" %}
+
+### Getting Your Fork On!
+
+You're first step in the process will be going to the address of the repository that you will want to fork. For this example it will be the one and only rockstar, ninja, guru, [insert more cliche buzzwords], awesome [community Firehose Project Jekyll site](https://github.com/FirehoseCommunity/community.thefirehoseproject.com)!
+
+`https://github.com/FirehoseCommunity/community.thefirehoseproject.com`
+
+Pretty simple so far, now we're going to click on that wonderful little "fork" button. If it gives you an option to which organization you want to fork then you will just want to make sure you select your personal page.
+
+![Image of Fork button](https://github-images.s3.amazonaws.com/help/bootcamp/Bootcamp-Fork.png)
+
+Next we will want to move to our now forked repository, get the URL for the cloning process, and do a quick copy to our clipboard.
+
+![Image of Clone button](https://help.github.com/assets/images/help/repository/clone-repo-clone-url-button.png)
+
+Moving on we will want to open up our terminal, move to the directory that we want to host our files locally and run a git clone command.
+
+Example (yours may look different):
+
+`git clone git@github.com:FirehoseCommunity/community.thefirehoseproject.com.git`
+
+Now we will want to move into our newly created directory. From here what you want to do it make sure everything is setup properly and that the Jekyll server serves up the data.
+
+You can do this by running (mileage may vary due to differing environments).
+
+`jekyll serve`
```

Comment:
Nah.  If you fixed it in the config (makes sense), then we don't need to confuse people.  It also would cause an inconsistency with the post and the video, which isn't needed doesn't make sense.